src/main/kotlin/Command/commands/ChooseNoodlesCommand.kt

```diff
@@ -0,0 +1,19 @@
+package Command.commands
+
+import Command.base.Cart
+import Command.base.Command
+
+class ChooseNoodlesCommand(val cart : Cart) : Command {
```

Comment:
Not, it's fine, you are right, the command have a receiver that is cart object 👍